src/main/java/org/myrobotlab/service/interfaces/AxisListener.java

```diff
@@ -0,0 +1,30 @@
+package org.myrobotlab.service.interfaces;
+
+import org.myrobotlab.framework.interfaces.NameProvider;
+import org.myrobotlab.service.data.AxisData;
+
+public interface AxisListener extends NameProvider {
```

Comment:
this should have an attachAxisPublisher and AxisPublisher should be able to attachAxisListener

Reply:
fixed